gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Add feature id tracking for credentials.
```

Comment:
Could use similar wording that we had previously when we update business metrics, i.e. [example 1](https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-core/CHANGELOG.md?plain=1#L191) and [example 2](https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-core/CHANGELOG.md?plain=1#L323)